[2025-08-19 03:09:32,942] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-19T03:09:31.215439+00:00 [queued]>
[2025-08-19 03:09:32,960] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-19T03:09:31.215439+00:00 [queued]>
[2025-08-19 03:09:32,961] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-19 03:09:32,961] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-19 03:09:32,961] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-19 03:09:32,987] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): fetch_and_send_weather> on 2025-08-19 03:09:31.215439+00:00
[2025-08-19 03:09:32,994] {standard_task_runner.py:52} INFO - Started process 11414 to run task
[2025-08-19 03:09:32,999] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'fetch_and_send_weather', 'manual__2025-08-19T03:09:31.215439+00:00', '--job-id', '319', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpwki6rby8', '--error-file', '/tmp/tmpgtoqtvht']
[2025-08-19 03:09:33,001] {standard_task_runner.py:80} INFO - Job 319: Subtask fetch_and_send_weather
[2025-08-19 03:09:33,084] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.fetch_and_send_weather manual__2025-08-19T03:09:31.215439+00:00 [running]> on host a4b544519f2b
[2025-08-19 03:09:33,215] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=fetch_and_send_weather
AIRFLOW_CTX_EXECUTION_DATE=2025-08-19T03:09:31.215439+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-08-19T03:09:31.215439+00:00
[2025-08-19 03:09:33,216] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-19 03:09:33,217] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /opt/***/scripts && python3 kafka/producer.py']
[2025-08-19 03:09:33,241] {subprocess.py:85} INFO - Output:
[2025-08-19 03:09:35,176] {subprocess.py:92} INFO - Envoi vers Kafka : {'temperature': 25.4, 'humidity': 90, 'windSpeed': 0.9, 'cloudCover': 100, 'rainIntensity': 0, 'timestamp': '2025-08-19T03:09:00Z'}
[2025-08-19 03:09:35,245] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-19 03:09:35,293] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=fetch_and_send_weather, execution_date=20250819T030931, start_date=20250819T030932, end_date=20250819T030935
[2025-08-19 03:09:35,344] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-19 03:09:35,410] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
